dev-lang/rust/rust-1.27.0.recipe

```diff
@@ -0,0 +1,236 @@
+srcGitRev14="eebda16e4b45f2eed4310cf7b9872cc752278163"
+srcGitRev15="a19ca1cd91cf97777af8268a6136bd2e4648e189"
+srcGitRev16="1742229ebb7843a65c05ee495d8de5366fcc5567"
+SOURCE_URI="https://github.com/rust-lang/rust/archive/$portVersion.tar.gz"
+SOURCE_URI_2="https://github.com/rust-lang/llvm/archive/$srcGitRev2.tar.gz"
+SOURCE_URI_3="https://github.com/rust-lang/rust-installer/archive/$srcGitRev3.tar.gz"
+SOURCE_URI_4="https://github.com/rust-lang/libc/archive/$srcGitRev4.tar.gz"
+SOURCE_URI_5="https://github.com/rust-lang-nursery/nomicon/archive/$srcGitRev5.tar.gz"
+SOURCE_URI_6="https://github.com/rust-lang/cargo/archive/$cargoVersion.tar.gz"
+SOURCE_URI_7="https://github.com/rust-lang-nursery/reference/archive/$srcGitRev7.tar.gz"
+SOURCE_URI_8="https://github.com/rust-lang/book/archive/$srcGitRev8.tar.gz"
+SOURCE_URI_9="https://github.com/rust-lang-nursery/rls/archive/$srcGitRev9.tar.gz"
+SOURCE_URI_10="https://github.com/rust-lang-nursery/compiler-builtins/archive/$srcGitRev10.tar.gz"
+SOURCE_URI_11="https://github.com/rust-lang/compiler-rt/archive/$srcGitRev11.tar.gz"
+SOURCE_URI_12="https://github.com/rust-lang-nursery/rustfmt/archive/$srcGitRev12.tar.gz"
+SOURCE_URI_13="https://github.com/solson/miri/archive/$srcGitRev13.tar.gz"
+SOURCE_URI_14="https://github.com/rust-lang/rust-by-example/archive/$srcGitRev14.tar.gz"
+SOURCE_URI_15="https://github.com/rust-lang-nursery/stdsimd/archive/$srcGitRev15.tar.gz"
+SOURCE_URI_16="https://github.com/rust-lang-nursery/rust-clippy/archive/$srcGitRev16.tar.gz"
+SOURCE_FILENAME="rust-$portVersion.tar.gz"
+CHECKSUM_SHA256="0bbca54761be5302efc6e6fdb5f4dab4e0dec24458ca7c13538b7259885f9457"
+CHECKSUM_SHA256_2="ba1578c156e8f10a9330f8d2b8514987c74a2babaea845ac4c4a1d2d5ef379ec"
+CHECKSUM_SHA256_3="34fd6dd8f8e5943be3ab022cd7d0ca792ea2cfe6fb21e06d6054134fc6441892"
+CHECKSUM_SHA256_4="aa083022606be28922dfd94606dc27da11f9f397ed732d93fd70fd6a8fa82594"
+CHECKSUM_SHA256_5="abf8866307d51259e55fef461e7954d05cf3169001069dc9ed4ab5eb40a73abd"
+CHECKSUM_SHA256_6="62f4269af6cd168c91a6b10ef845e6dc0f1c211605bd13d155a6a001ade026a1"
+CHECKSUM_SHA256_7="9e54405f4a6c487cba04d2e25e7026b86581038a88741ce5757b38ddfab4cb07"
+CHECKSUM_SHA256_8="a18b090a3a29313a9e68dbef0ca4a5e0ecb36d9eb0f244a1e6cbd33989546989"
+CHECKSUM_SHA256_9="5ca89b25cf636b018cb1e9862cf2272e1295874c712db23178e36a3b2dded892"
+CHECKSUM_SHA256_10="cebece5392d3990a4ed06af378f1e44daea0c30d1d348d1cefb7dae9a4f56bbb"
+CHECKSUM_SHA256_11="7384b3592a4b05857fbf4e54ff5f0ac17bb8bd982b428f348ae2cc4afd9a552c"
+CHECKSUM_SHA256_12="9708789d175ce1dd43e1623c1373629d26d583022e9e826d768d9270c4304f36"
+CHECKSUM_SHA256_13="7ab3a7345cc98e96a223b92653374fe1cae2c81f90afb745e933f02eacbf7390"
+CHECKSUM_SHA256_14="be12b7ba065470a046f7e2ba3f76abd3fd6028b24393eba292d024de8164b630"
+CHECKSUM_SHA256_15="ddfb9831e8cb439545ce8180a958e57877a0cd9fc3f54305b38679f2b0dd2f68"
+CHECKSUM_SHA256_16="e5b04a1dfeab2d89fc5ee4509711c381bcdca53586b73a999102e14ac8748c23"
+SOURCE_DIR="rust-$portVersion"
```

Comment:
```diff
-SOURCE_DIR="rust-$portVersion"
```
(We can remove this line because haikuporter already sets this as the default value.)